File: week6/mario.py
from cs50 import *

def main():
    while True:
        height = get_int("Height: ")
        if height > 0 and height < 9:
            break
    for i in range(1,height + 1):
        spaces = height - i

        print(" " * spaces, end="")
        print("#" * i, end="")
        print("  ", end="")
        print("#" * i)


if __name__ == "__main__":
    main()

File: pset1/pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int h = get_int("Height ");
    while (h<1 || h >8) {
    h = get_int("Height ");
    if(h>= 1 && h <=8){
        break;
    }
    }
    for(int i = 1; i <= h ; i++){
        for (int left_spaces = h - i; left_spaces > 0; left_spaces--)
        {
            printf(" ");
        }

        for (int lhashes = 1; lhashes <= i; lhashes++)
        {
            printf("#");
        }

        printf("  ");

        for (int rhashes = 1; rhashes <= i; rhashes++)
        {
            printf("#");
        }


        printf("\n");

    }
}